.circleci/config.yml

```diff
@@ -4,7 +4,7 @@ jobs:
 
   build_and_test:
     docker:
-      - image: circleci/node:8.11.2@sha256:495a13db597b5306f05ae728489fedb9faa4defbac24ca0e00e8cc1eca95396d
+      - image: circleci/node:10.18.1@sha256:6bb2f0665d1d46a963ffabff6334deb363a92e6bcfaaa6753b9d0ea3084680fd
```

Comment:
I got this information from [this](https://hub.docker.com/r/circleci/node/tags?page=1&name=10.18.1)